lua/terrortown/entities/roles/sidekick/shared.lua

```diff
@@ -49,8 +50,9 @@ hook.Add("TTTUlxDynamicRCVars", "TTTUlxDynamicSikiCVars", function(tbl)
 
 	table.insert(tbl[ROLE_SIDEKICK], {

```

Comment:
please replace this with a dropdown menu like the jester: 

https://github.com/TTT-2/ttt2-role_jes/blob/master/lua/terrortown/entities/roles/jester/shared.lua#L45-L60

Reply:
Fixed with Commit 3836efed7a0e5d242d1f742f7717f3286ed07809 "changed slider to combobox in ulx convar menu".